### ISSUE TITLE
E756: Spell checking is not possible on insert mode

### ISSUE THREAD

**Issue by otavioschwanck** [neoclide/coc.nvim#3700]
## versions

vim version: NVIM v0.7.0-dev+1245-gf291a0339
node version: v14.18.1
coc.nvim version: 0.0.80-47c4dee3 2022-03-13 01:40:13 +0800
coc.nvim directory: /Users/otavio/.local/share/nvim/site/pack/packer/start/coc.nvim
term: tmux
platform: darwin

## Log of coc.nvim

2022-03-13T01:52:18.694 INFO (pid:10689) [plugin] - coc.nvim initialized with node: v14.18.1 after 280ms
2022-03-13T01:52:18.696 INFO (pid:10689) [attach] - receive notification: highlight []
2022-03-13T01:52:23.357 INFO (pid:10689) [attach] - receive notification: installExtensions []
2022-03-13T01:52:23.560 INFO (pid:10689) [attach] - receive notification: highlight []
2022-03-13T01:52:25.124 INFO (pid:10689) [attach] - receive notification: highlight []
2022-03-13T01:52:27.036 INFO (pid:10689) [services] - registered service "solidity"
2022-03-13T01:52:27.036 INFO (pid:10689) [services] - Solidity Language Server state change: stopped => starting
2022-03-13T01:52:27.037 INFO (pid:10689) [attach] - receive notification: highlight []
2022-03-13T01:52:27.059 INFO (pid:10689) [core-watchman] - watchman watching project: /Users/otavio/Projetos/nest-onchain
2022-03-13T01:52:27.603 INFO (pid:10689) [completion-complete] - 2 results from: around,snippets
2022-03-13T01:52:29.366 INFO (pid:10689) [attach] - Request action: snippetCheck [ 1, 1 ]
2022-03-13T01:52:29.524 INFO (pid:10689) [attach] - Request action: snippetCheck [ 1, 1 ]
2022-03-13T01:52:30.667 INFO (pid:10689) [completion-complete] - 2 results from: around,snippets
2022-03-13T01:52:31.930 INFO (pid:10689) [completion-complete] - 2 results from: around,snippets
2022-03-13T01:52:32.041 INFO (pid:10689) [services] - Solidity Language Server state change: starting => running
2022-03-13T01:52:32.043 INFO (pid:10689) [services] - service solidity started
2022-03-13T01:52:33.907 INFO (pid:10689) [completion-complete] - 3 results from: around,snippets,solidity
2022-03-13T01:52:42.240 INFO (pid:10689) [attach] - receive notification: highlight []
2022-03-13T01:52:45.803 INFO (pid:10689) [attach] - receive notification: highlight []
2022-03-13T01:52:47.872 INFO (pid:10689) [attach] - Request action: extensionStats []
2022-03-13T01:52:54.769 INFO (pid:10689) [attach] - receive notification: uninstallExtension [ 'coc-jedi' ]
2022-03-13T01:52:55.160 INFO (pid:10689) [attach] - receive notification: highlight []
2022-03-13T01:52:56.033 INFO (pid:10689) [attach] - receive notification: highlight []
2022-03-13T01:52:56.682 INFO (pid:10689) [completion-complete] - 3 results from: around,snippets,solidity
2022-03-13T01:52:57.740 INFO (pid:10689) [completion-complete] - 3 results from: around,snippets,solidity
2022-03-13T01:53:01.315 INFO (pid:10689) [attach] - Request action: extensionStats []
2022-03-13T01:53:06.354 INFO (pid:10689) [attach] - receive notification: uninstallExtension [ 'coc-pyls' ]
2022-03-13T01:53:06.564 INFO (pid:10689) [attach] - receive notification: highlight []
2022-03-13T01:53:12.181 INFO (pid:10689) [attach] - receive notification: uninstallExtension [ '*' ]
2022-03-13T01:53:12.385 INFO (pid:10689) [attach] - receive notification: highlight []
2022-03-13T01:53:13.448 INFO (pid:10689) [attach] - receive notification: highlight []
2022-03-13T01:53:13.655 INFO (pid:10689) [attach] - receive notification: highlight []
2022-03-13T01:53:15.471 INFO (pid:10689) [attach] - Request action: extensionStats []
2022-03-13T01:53:21.100 INFO (pid:10689) [attach] - receive notification: highlight []
2022-03-13T01:53:21.717 INFO (pid:10689) [attach] - receive notification: highlight []
2022-03-13T01:53:22.354 INFO (pid:10689) [attach] - receive notification: highlight []
2022-03-13T01:53:22.593 INFO (pid:10689) [completion-complete] - 3 results from: around,snippets,solidity
2022-03-13T01:53:24.742 INFO (pid:10689) [attach] - receive notification: highlight []
2022-03-13T01:53:45.877 INFO (pid:10689) [attach] - receive notification: highlight []
2022-03-13T01:53:46.506 INFO (pid:10689) [attach] - Request action: snippetCheck [ 1, 1 ]
2022-03-13T01:53:48.796 INFO (pid:10689) [completion-complete] - 3 results from: around,snippets,solidity
2022-03-13T01:53:52.686 INFO (pid:10689) [completion-complete] - 3 results from: around,snippets,solidity
2022-03-13T01:54:27.902 INFO (pid:10689) [attach] - receive notification: highlight []
2022-03-13T01:54:29.375 INFO (pid:10689) [attach] - receive notification: showInfo []
2022-03-13T01:54:29.598 INFO (pid:10689) [attach] - receive notification: highlight []
2022-03-13T01:54:32.495 INFO (pid:10689) [attach] - receive notification: highlight []
2022-03-13T01:54:38.888 INFO (pid:10689) [attach] - receive notification: highlight []
2022-03-13T01:54:39.697 INFO (pid:10689) [attach] - receive notification: highlight []
2022-03-13T01:54:40.611 INFO (pid:10689) [attach] - receive notification: highlight []
2022-03-13T01:54:41.457 INFO (pid:10689) [attach] - receive notification: highlight []
2022-03-13T01:54:42.203 INFO (pid:10689) [attach] - receive notification: highlight []
2022-03-13T01:54:43.544 INFO (pid:10689) [attach] - receive notification: highlight []
2022-03-13T01:54:45.386 INFO (pid:10689) [completion-complete] - 3 results from: around,snippets,solidity
2022-03-13T01:54:47.649 INFO (pid:10689) [attach] - receive notification: highlight []
2022-03-13T01:54:49.346 INFO (pid:10689) [attach] - receive notification: showInfo []

## Describe the bug

In insert mode, when i try to type `pendingAsset` it throws `E756: Spell checking is not possible`.

it throws when try to type gAs.

It only occurs on coc-pyright, coc-solidity and coc-typescript.  But on coc-solargraph (ruby) not for some reason.

nvim on HEAD (mac-os)

## Reproduce the bug

on a python file, type gAss on insert mode.

- Create file `mini.vim` with：

```vim
lua <<LUA
return require('packer').startup(function()
  use {'neoclide/coc.nvim', branch = 'release'}
end)
LUA

let g:coc_global_extensions = ['coc-solidity', 'coc-snippets', 'coc-pyright']
```
  
  Please, use nvim for the test.

gif:
![Kapture 2022-03-13 at 02 03 12](https://user-images.githubusercontent.com/9551316/158045934-0a874599-f5b7-4b5a-bde2-756fc863cf3a.gif)

**Comment by chemzqm** [comment]
Fixed by ed45a07a

**Comment by otavioschwanck** [comment]
Holy shit, that was fast, THANK YOU <3
